packages/administration/templates/content/product/visibilityItems.html.twig

```diff
@@ -30,10 +47,16 @@
             <div class="col">
                 {% if isVisible %}
                     <a href="{{ url }}" class="" target="_blank">
```

Comment:
this was probably my error from earlier, but empty class does not make sense imho